### ISSUE TITLE
Qt6 musl support - gives runtime error

### ISSUE THREAD

**Issue by creepinson** [slint-ui/slint#960]
I am trying to use the backend-qt feature with Arch Linux, and I am compiling my code for x86_64-unknown-linux-musl. However, it gives the following error even though I have the qt6 libraries installed.

```bash
Running `/home/theo/dev/theos/core/target/x86_64-unknown-linux-musl/debug/packa-gui`
thread 'main' panicked at 'The Qt backend needs Qt', /home/theo/.cargo/registry/src/github.com-1ecc6299db9ec823/i-slint-backend-qt-0.2.0/lib.rs:110:9
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

Any ideas on how I can fix this?

For any other arch linux users, I installed the `qt6` package which contains many base qt6 packages so I'd assume itd work fine.

Also, I cannot seem to use the default features (with x11) either due to servo-fontconfig-sys v5.1.0 not supporting musl (it uses pkgconfig which means I'd need to setup a whole cross compiling pkgconfig toolchain...)

**Comment by creepinson** [comment]
Well I tried to run winit with egui and it gave dynamic loading not supported, which is potentially why this won't work either.

**Comment by ogoffart** [comment]
Is qmake on your path?
If not, you might need to set the QMAKE env variable or something. I believe the arch packages might have renamed qmake to qmake6 or something like that.
See https://docs.rs/qttypes/latest/qttypes/#finding-qt and https://github.com/slint-ui/slint/blob/master/docs/install_qt.md

**Comment by ogoffart** [comment]
Building qttype should show warning

**Comment by creepinson** [comment]
Seems like I have a qmake6 executable but not a qmake one as you mentioned. It did not seem to show a warning but when I prepended the QMAKE variable to cargo run it worked. Musl gave a ton of glibc linker errors so I'm going to try to use alpine linux in docker to build it, but for now it works fine with qt6 and glibc 👍🏼 

```
QMAKE="/usr/bin/qmake6" cargo run
```

I think  this can be closed as the main compilation problem has been solved.
